src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -321,6 +332,19 @@ public static boolean isItemSimilar(@Nullable ItemStack item, @Nullable ItemStac
         }
     }
 
+    private static @Nullable String getMetaId(ItemMeta itemMeta) {
```

Comment:
Not sure why this doesn't return an Optional. Makes this code very weird. Not sure we even need this method at all tbh